packages/core/content-releases/admin/src/pages/ReleasesSettingsPage.tsx

```diff
@@ -85,13 +95,9 @@ export const SettingsPage = () => {
       <Page.Main aria-busy={isLoadingSettings} tabIndex={-1}>
         <Form
           method="PUT"
-          initialValues={
-            data?.data.defaultTimezone
-              ? data.data
-              : {
-                  defaultTimezone: null,
-                }
-          }
+          initialValues={{
+            defaultTimezone: data?.data.defaultTimezone || '',
```

Comment:
If the API defaultTimezone is null, and then we set the defaultTimezone form value to '', wouldn't we be overriding the value to be an empty string after saving the settings?